frontend/packages/console-dynamic-plugin-sdk/CHANGELOG-core.md

```diff
@@ -18,6 +18,7 @@ table in [Console dynamic plugins README](./README.md).
 - Begin alignment of plugin SDK types with `@openshift/dynamic-plugin-sdk` ([CONSOLE-3769], [#15509])
 - Add optional `fetch` property to extension `console.dashboards/overview/health/url` ([CONSOLE-4796], [#15526])
 - Add optional `infrastructure` parameter to `PrometheusHealthHandler` type ([CONSOLE-4796], [#15526])
+- Allow `K8sResourceKind` in `TopologyDataObject` and `TopologyResourcesObject` type ([CONSOLE-4840], [#15699])
```

Comment:
Required to fix 57 type errors on knative and topology unit tests. Not a breaking change because all extra properties of `K8sResourceKind` are optional